### PR TITLE
rename weights format pytorch_script to torchscript

### PR DESCRIPTION
**Pull request by FynnBe**
and fix unet2d_braod_nuclei version conversion tests (were broken by new maintainers field)

### CODE REVIEW

File: bioimageio/spec/model/v0_4/raw_nodes.py [hunk 1]
```diff
@@ -48,6 +46,9 @@
 PreprocessingName = PreprocessingName
 
 FormatVersion = Literal["0.4.0"]  # newest format needs to be last (used in __init__.py)
+WeightsFormat = Literal[
+    "pytorch_state_dict", "torchscript", "keras_hdf5", "tensorflow_js", "tensorflow_saved_model_bundle", "onnx"
```
Comment (constantinpape) [inline bioimageio/spec/model/v0_4/raw_nodes.py:50]: Does the order here matter? Does it determine the weight preferences in core?

File: scripts/generate_weight_formats_overview.py [hunk 2]
```diff
@@ -14,9 +14,9 @@
 
 # defaults for transition period
 consumer_defaults = {
-    "ilastik": ["pytorch_state_dict", "onnx", "pytorch_script"],
+    "ilastik": ["pytorch_state_dict", "onnx", "torchscript"],
     "zero": ["keras_hdf5"],
-    "deepimagej": ["pytorch_script", "tensorflow_saved_model_bundle"],
+    "deepimagej": ["tensorflow_saved_model_bundle"],
```
Comment (constantinpape) [inline scripts/generate_weight_formats_overview.py:19]: We shouldn't drop `torchscript` from deepimageJ, we need to support it there in order to demonstrate compatibility with ilastik.
(They may need to update for this one though, but I think that's ok as it shouldn't concern most of the older models.)

**Review by constantinpape — COMMENTED** [review]
Looks good overall, but we shouldn't remove torchscript from deepIJ

**Comment by constantinpape** [comment]
Test failures are only against models on main, so this shouldn't matter.

File: bioimageio/spec/model/v0_4/raw_nodes.py (hunk 1 above)
Comment (FynnBe) [reply bioimageio/spec/model/v0_4/raw_nodes.py:50]: no, core has its own list (as it also may have a shorter list). It needs a separate update here: https://github.com/bioimage-io/core-bioimage-io-python/blob/1200b5a9d9b51430217c1adea88929f880b4e038/bioimageio/core/prediction_pipeline/_model_adapters/_model_adapter.py#L10

**Comment by constantinpape** [comment]
All relevant tests are passing, I am gonna merge it.

**Pull request merged**